pkgs/top-level/all-packages.nix

```diff
@@ -12771,6 +12767,10 @@ in
 
     libopenshot-audio = callPackage ../applications/video/openshot-qt/libopenshot-audio.nix { };
 
+    libqglviewer = callPackage ../development/libraries/libqglviewer {
```

Comment:
Why not call `libsForQt5.callPackage` directly here?

Reply:
Why would I do that? The package is being defined within the `mkLibsForQt5` function, which allows building the package with multiple versions of Qt5 (and helps prevent accidental mixing of versions), while `libsForQt5` is the package set for the latest version of Qt5.

Reply:
Oh my mistake, I failed to notice the move inside `mkLibsForQt5`